pkgs/development/libraries/arrayfire/default.nix

```diff
@@ -1,91 +1,177 @@
-{ lib
-, stdenv
-, fetchFromGitHub
+{ blas
+, boost
 , cmake
-, pkg-config
-, opencl-clhpp
-, ocl-icd
+, config
+, cudaPackages
+, fetchFromGitHub
 , fftw
 , fftwFloat
-, blas
+, fmt_9
+, forge
+, freeimage
+, git
+, gtest
 , lapack
-, boost
-, mesa
-, libGLU
+, lib
 , libGL
-, freeimage
+, mesa
+, ocl-icd
+, opencl-clhpp
+, pkg-config
 , python3
-, clfft
-, clblas
-, doxygen
-, buildDocs ? false
-, config
-, cudaSupport ? config.cudaSupport
-, cudatoolkit
-, darwin
+, span-lite
+, stdenv
+, doCheck ? false
+, withCPU ? true
+, withCuda ? config.cudaSupport
```

Comment:
I notice this changes the attribute name `cudaSupport -> withCuda` which people may have already been using. I see how the new naming is more consistent (`withCPU`, `withOpenCL`, so `withCuda`), but I suspect that `cudaSupport` is used more often across nixpkgs, and it's the status quo name since the expression has already been using it.

Personally, I don't mind renaming and I'll approve the PR even if you keep this extra change. I do think, however, there are more reasons to keep the old name than not to